src/main/kotlin/io/github/mojira/arisa/infrastructure/config/Arisa.kt

```diff
@@ -9,191 +9,73 @@ object Arisa : ConfigSpec() {
-                
-                You are currently using a *non-authorized* version of Minecraft. If you wish to purchase the full game, please visit the [Minecraft Store|https://www.minecraft.net/store/minecraft-java-edition].
-                We will not provide support for pirated versions of the game, these versions are modified and may contain malware.
-                
-                *Quick Links*:
-                📓 [Issue Guidelines|https://bugs.mojang.com/projects/MC/summary] -- 💬 [Community Support|https://discord.gg/58Sxm23] -- 📧 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- 📖 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki]
-                
-                ~{color:#888}-- I am a bot. This action was performed automagically! Please report any issues in [Discord|https://discordapp.com/invite/rpCyfKV] or [Reddit|https://www.reddit.com/r/Mojira/]{color}~
-                """.trimIndent()
-            )
-            val piracySignatures by optional(
-                listOf(
-                    "Minecraft Launcher null",
-                    "Bootstrap 0",
-                    "Launcher: 1.0.10  (bootstrap 4)",
-                    "Launcher: 1.0.10  (bootstrap 5)",
-                    "Launcher 3.0.0",
-                    "Launcher: 3.1.0",
-                    "Launcher: 3.1.1",
-                    "Launcher: 3.1.4",
-                    "1.0.8",
-                    "uuid sessionId",
-                    "auth_access_token",
-                    "windows-\${arch}",
-                    "keicraft",
-                    "keinett",
-                    "nodus",
-                    "iridium",
-                    "mcdonalds",
-                    "uranium",
-                    "divinity",
-                    "gemini",
-                    "mineshafter",
-                    "Team-NeO",
-                    "DarkLBP",
-                    "Launcher X",
-                    "PHVL",
-                    "Pre-Launcher v6",
-                    "LauncherFEnix",
-                    "TLauncher",
-                    "Titan"
-                )
-            )
+            val message by optional("", description = "The message that is posted when this module succeeds.")
+            val piracySignatures by optional(emptyList<String>(), description = "Signatures that indicate a pirated version of Minecraft. Default is no signatures.")
         }
 
         object RemoveTriagedMeqs : ModuleConfigSpec() {
-            val meqsTags by optional(listOf("MEQS_WAI", "MEQS_WONTFIX"))
-            val removalReason by optional("Ticket has been triaged.")
+            val meqsTags by optional(emptyList<String>(), description = "List of tags that should be removed by the bot when an issue is triaged.")
+            val removalReason by optional<String?>(null, description = "Reason Arisa should add to the edited comment for removing the tag. Default is no reason.")
```

Comment:
I wouldnt have null until we need it in any message, it seems more unclean(we try to avoid null as much as possible)

Reply:
null is the default value. If no reason is set in the config, it would be weird to have "Removal Reason: <blank>", so when it's null the module does not post `Removal Reason`

Reply:
Yeah, I would make it required is what I mean